int32 seq_len
string traj_name
float64 v_input
float64[] trajectory
float64[] dt
float64[] inputs
